daml-foundations/daml-ghc/package-database/util.bzl

```
@@ -77,14 +77,18 @@ def _daml_package_rule_impl(ctx):
         progress_message = "Compiling " + name + ".daml to daml-lf " + ctx.attr.daml_lf_version,
         command = """
       set -eou pipefail
-      mkdir -p tmp_db
-      tar xf {db_tar} -C tmp_db --strip-components 1
-      mkdir -p tmp_db/{daml_lf_version}
+      PACKAGEDB="package_db_for_{name}"
+      # Since we don’t have sandboxing on Windows, that directory might
```

```suggestion
      # Since we don't have sandboxing on Windows, that directory might
```

Smart quotes seem like a terrible idea here